project/Dependencies.scala

```diff
@@ -13,8 +13,8 @@
 import sbt._
 
 object Dependencies {
-  val catsEffect    = "org.typelevel"      %% "cats-effect"          % "1.3.1"
-  val cats          = "org.typelevel"      %% "cats-core"            % "1.6.1"
+  val catsEffect    = "org.typelevel"      %% "cats-effect"          % "2.0.0"
+  val cats          = "org.typelevel"      %% "cats-core"            % "2.0.0"
```

Comment:
Any reason to not use 2.1.0?

Reply:
2.1.0 is not published for Scala 2.11, so moving forward requires dropping support for that version

Reply:
I think we should be ready to make this sacrifice. Many downstream libraries (like SCE) already dropped 2.11.

Reply:
ok, done